types/relay-runtime/lib/handlers/RelayDefaultMissingFieldHandlers.d.ts

```diff
@@ -1,3 +1,3 @@
 import { MissingFieldHandler } from '../store/RelayStoreTypes';
 
-export const missingViewerFieldHandler: MissingFieldHandler;
+export const DefaultMissingFieldHandlers: MissingFieldHandler[];
```

Comment:
2. …so this should really be something like:

```suggestion
declare const DefaultMissingFieldHandlers: MissingFieldHandler[];
export default DefaultMissingFieldHandlers
```